dev/AnimatedVisualPlayer/AnimatedVisualPlayer.cpp

```diff
@@ -911,7 +911,7 @@ void AnimatedVisualPlayer::UpdateContent()
         auto to = 1;
         auto looped = true;
         // NOTE: If !IsAnimatedVisualLoaded() then this is a no-op.
-        PlayAsync(from, to, looped);
+        auto ignore = PlayAsync(from, to, looped);
```

Comment:
I have to say, I don't LOVE this rule. I would rather nodiscard be added to functions where it is critical not to ignore the return value. But rules is rules I suppose.

Reply:
It is annoying but in a lot of these cases it's the same as C# -- you forgot to "await" something async and you should be explicit that you are ignoring it.